WooCommerce/Classes/ViewRelated/Orders/Payment Methods/PaymentMethodsViewModel.swift

```diff
@@ -338,13 +338,33 @@ private extension PaymentMethodsViewModel {
     ///
     @MainActor
     func markOrderAsPaid() async throws {
-        try await withCheckedThrowingContinuation { continuation in
-            stores.dispatch(OrderAction.updateOrderStatus(siteID: siteID, orderID: orderID, status: .completed) { error in
-                guard let error else {
-                    return continuation.resume(returning: ())
+        try await withCheckedThrowingContinuation { (continuation: CheckedContinuation<Void, Error>) in
+            guard let order = ordersResultController.fetchedObjects.first else {
+                DDLogError("⛔️ Order not found, can't mark order as paid.")
```

Comment:
Just for easier debugging, we could interpolate the orderID here:
```suggestion
                DDLogError("⛔️ Order \(orderID) not found, can't mark order as paid.")
```